server/mscalendar/availability.go

```diff
@@ -63,9 +71,84 @@ func (m *mscalendar) SyncAll() (string, *StatusSyncJobSummary, error) {
 		return "", &StatusSyncJobSummary{}, errors.Wrap(err, "not able to load the users from user index")
 	}
 
+	err = m.Filter(withSuperuserClient)
+	// Allow processing the daily summary using individual credentials for remotes that doesn't allow
+	// "superuser" access
+	if errors.Is(err, remote.ErrSuperUserClientNotSupported) {
+		return m.syncUsersIndividually(userIndex)
+	}
+	if err != nil {
+		return "", &StatusSyncJobSummary{}, errors.Wrap(err, "not able to filter the super user client")
+	}
+
 	return m.syncUsers(userIndex)
 }
 
+func (m *mscalendar) syncUsersIndividually(userIndex store.UserIndex) (string, *StatusSyncJobSummary, error) {
```

Comment:
What do you think about trying to de-duplicate some code between `syncUsers` and  `syncUsersIndividually`? Probably good to have a unit test for this as well